qa/tasks/persistent_write_log_cache_thrash.py

```diff
@@ -0,0 +1,102 @@
+"""
+persistent write log cache thrash task
+"""
+import contextlib
+import logging
+import random
+
+from teuthology import misc as teuthology
+from teuthology import contextutil
+from teuthology.orchestra import run
+
+DEFAULT_NUM_ITERATIONS = 20
+IO_PATTERNS = ("full-seq", "rand")
+IO_SIZES = ('4K', '16K', '128K', '1024K')
+
+log = logging.getLogger(__name__)
+
+@contextlib.contextmanager
+def thrashes_rbd_bench_on_persistent_cache(ctx, config):
+    """
+    thrashes rbd bench on persistent write log cache.
+    It can test recovery feature of persistent write log cache.
+    """
+    log.info("thrashes rbd bench on persistent write log cache")
+
+    client, client_config = list(config.items())[0]
+    (remote,) = ctx.cluster.only(client).remotes.keys()
+    client_config = client_config if client_config is not None else dict()
+    image_name = client_config.get('image_name', 'testimage')
+    cache_path = client_config.get('rbd_persistent_cache_path', '/home/ubuntu/cephtest/rbd-pwl-cache')
+    num_iterations = client_config.get('num_iterations', DEFAULT_NUM_ITERATIONS)
+    ls_command = '"$(ls {})"'.format(cache_path)
+
+    for i in range(num_iterations):
+        log.info("start rbd bench")
+        # rbd bench could not specify the run time so set a large enough test size.
+        remote.run(
+            args=[
+                'rbd', 'bench',
+                '--io-type', 'write',
+                '--io-pattern', random.choice(IO_PATTERNS),
+                '--io-size', random.choice(IO_SIZES),
+                '--io-total', '102400M',
+                image_name,
+                ],
+            wait=False,
+        )
+        # Wait a few seconds for the rbd bench process to run
+        # and complete the pwl cache initialization
+        remote.run(
+            args=[
+                'sleep', '10',
+                ]
+        )
+        proc = remote.run(
+            args=[
+                'rbd', 'status', image_name
+                ],
+            wait=False,
+            stdout=run.PIPE
+        )
+        assert b"Persistent cache state" in proc.stdout.read(), "persistent cache enable failed."
+
+        log.info("sleep...")
+        remote.run(
+            args=[
+                'sleep', '{live_time}'.format(live_time=random.randint(10,60)),
+                ]
+        )
+        log.info("rbd bench crash.")
+        remote.run(
+            args=[
+                'killall', '9', 'rbd', run.Raw('||'), 'true',
```

Comment:
```suggestion
                'killall', '-9', 'rbd', run.Raw('||'), 'true',
```

Reply:
Also, you can pass `check_status=False` to `remote.run()` instead of bothering with ` || true`.